### PR TITLE
[IMPROVE] Decrease log level of trace Everything OK 0 requests from INFO to DEBUG #310

### PR DESCRIPTION
**Pull request by sanchit31**
1. The trace {"op":"OPER_STH_SERVER_LOG","time":"2016-04-07T02:11:55.606Z","lvl":"INFO","msg":"Everything OK, 0 requests attended in the last 60s interval"} is changed from log level INFO to DEBUG.

2. A new trace is added to count the number of requests processed with error:
"msg"="We have processed 0 requests with ERROR in the last 60s interval"

This pull request is created for the following issue:
[IMPROVE] Decrease log level of trace Everything OK 0 requests from INFO to DEBUG #310

### CODE REVIEW

File: config.js [hunk 1]
```diff
@@ -146,7 +146,10 @@ config.logging = {
     format: 'pipe',
     // The time in seconds between proof of life logging messages informing that the server is up and running normally.
     // Default value: "60"
-    proofOfLifeInterval: '60'
+    proofOfLifeInterval: '60',
+    // The time in seconds between proof of performance logging messages informing that the server is up and running normally.
+    // Default value: "60"
+    proofOfPerformanceInterval: '60'
```
Comment (fgalan) [inline config.js]: As far as I understand, this is not exactly a "proof of performance" but some statistics about request. Thus, I'd suggest to change the name, for instance "processedRequestLogStatisticsInterval". Code comment should be also adapted to the new name.

Comment (fgalan) [reply config.js]: Similar adaptation for PROOF_OF_PERFORMANCE_INTERVAL env var name.

File: lib/server/sthServer.js [hunk 2]
```diff
@@ -227,19 +232,39 @@ function getKPIs() {
     };
 }
 
+/**
+ * Returns the server PIs
```
Comment (fgalan) [inline lib/server/sthServer.js]: What is PIs here?

File: lib/configuration/sthConfiguration.js [hunk 3]
```diff
@@ -740,5 +740,29 @@ if (ENV.PROOF_OF_LIFE_INTERVAL && !isNaN(ENV.PROOF_OF_LIFE_INTERVAL)) {
             module.exports.PROOF_OF_LIFE_INTERVAL
     );
 }
+if (ENV.PROOF_OF_PERFORMANCE_INTERVAL && !isNaN(ENV.PROOF_OF_PERFORMANCE_INTERVAL)) {
```
Comment (fgalan) [inline lib/configuration/sthConfiguration.js]: The new variable should be described in the list at doc/manuals/running.md document.

File: config.js (hunk 1 above)
Comment (sanchit31) [reply config.js]: Made changes as per your suggestion. Fixed in 77610b5

File: lib/configuration/sthConfiguration.js (hunk 3 above)
Comment (sanchit31) [reply lib/configuration/sthConfiguration.js]: Added the new variable in the list at doc/manuals/running.md document. Fixed in 77610b5

File: lib/server/sthServer.js (hunk 2 above)
Comment (sanchit31) [reply lib/server/sthServer.js]: It's just a measurement value to return the number of processed requests. Named it along the lines of KPIs which returns the number of attended requests. Open to your suggestions for renaming it.

Comment (fgalan) [reply lib/server/sthServer.js]: getPI() -> getProcessedRequestWithErrorCount()
resetPI() -> resetProcessedRequestWithErrorCount()

A bit long, but I think that very descriptive. What do you think?

(In fact, I don't like too much getKPI() and resetKPI() for the other counter, but we can leave it as it is now)

Comment (sanchit31) [reply lib/server/sthServer.js]: I think it's a good suggestion. Just a minor change I'd like to have.
getProcessedRequestWithErrorCount() -> getProcessedRequestsWithErrorCount()
resetProcessedRequestWithErrorCount -> resetProcessedRequestsWithErrorCount()

Comment (fgalan) [reply lib/server/sthServer.js]: Ah, yes... I missed the 's' in 'Request**s**'.

Yes, of course :)

Comment (sanchit31) [reply lib/server/sthServer.js]: Fixed in c79c446

File: lib/server/sthServer.js [hunk 4]
```diff
@@ -59,6 +61,7 @@ function doStartServer(host, port, callback) {
 
     server.on('request-internal', function(request, event, tags) {
         if (tags.error) {
+        	processedRequestsWithError++;
```
Comment (AlvaroVega) [inline lib/server/sthServer.js]: Bad indentation?

Comment (fgalan) [reply lib/server/sthServer.js]: Yes. It seems so.

Comment (sanchit31) [reply lib/server/sthServer.js]: Corrected the indentation. Fixed in 5fe4c92

**Comment by AlvaroVega** [comment]
LGTM

**Pull request merged**